components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIManager.java

```diff
@@ -18,12 +18,12 @@
 
 package org.wso2.carbon.apimgt.api;
 
+import org.wso2.carbon.apimgt.api.dto.OrganizationDTO;
```

Comment:
shall we move this to model package?

Reply:
I think right place for Organization (in persistence package) is api package. we can move it later

Reply:
The OrganizationDTO just carries the data from the REST layer to the Persistence layer Nah?  Is it wrong to have that in the DTO section?

In the effort of merging the Organization (in persistence package) and this DTO, we can have it under the model package. Is this making sense?